src/components/AvatarShape.js

```diff
@@ -77,18 +85,23 @@ const ShapeOutline = styled(Link)`
 const AvatarShape = props => (
   <ShapeOutline shape={props.shape} bg={props.bg} {...props}>
     <A className={props.className} shape={props.shape}>
-      <Image src={props.src} />
+      <Image
+        src={props.src}
+        $greyscale={props.greyscale}
+      />
       <Shape shape={props.shape} />
     </A>
   </ShapeOutline>
 )
 
 AvatarShape.defaultProps = {
-  bg: 'blue.4'
+  bg: 'blue.4',
+  greyscale: false,
```

Comment:
Should this be conditional? I think you can make it greyscale by default, as per the current site, which would remove the need for conditional css and extra props

Reply:
The greyscale is turned off on hover so the gifs have colour:

| without greyscale | with greyscale |
|--------|--------|
| <img width="315" alt="Screenshot 2021-11-25 at 14 25 45" src="https://user-images.githubusercontent.com/1901935/143458764-03da5c0e-510c-4075-9182-7b036ac472ee.png"> | <img width="316" alt="Screenshot 2021-11-25 at 14 26 04" src="https://user-images.githubusercontent.com/1901935/143458751-c9a31de7-6c58-482f-953d-a36c3afc95d9.png">


I think ideally `AvatarShape` should be called `ShapeMask` and take a `children` prop for the avatar img so we don't have to pass the `greyscale` and `src` down (it's only used on this page) but I was wary of making bigger changes to this codebase (especially as I was only meant to be adding myself 😬)

Reply:
Yeah, I see the issue. I agree you'll need to forward a prop, so how about an alternative name of `hover` to give it more utility inside the component?

Reply:
Update: if you [try this](https://github.com/primer/primer.style/pull/279/files#r756989100), should fix this issue 👍